src/Assessor/Filter/AssessorFilters.php

```diff
@@ -63,6 +83,56 @@ public function getStatus(): ?string
         return $this->status;
     }
 
+    public function getCity(): ?string
+    {
+        return $this->city;
+    }
+
+    public function setCity($city): void
+    {
+        $this->city = $city;
+    }
+
+    public function getVotePlace(): ?string
+    {
+        return $this->votePlace;
+    }
+
+    public function setVotePlace($votePlace): void
+    {
+        $this->votePlace = $votePlace;
+    }
+
+    public function getCountry(): ?string
+    {
+        return $this->country;
+    }
+
+    public function setCountry(string $country): void
+    {
+        if (empty($country = strtoupper(trim($country)))) {
+            $this->country = null;
+
+            return;
+        }
+
+        if (!\in_array($country, array_keys($this->getCountries()), true)) {
+            throw new AssessorException(sprintf('Invalid country filter value given ("%s").', $country));
+        }
+
+        $this->country = trim($country);
+    }
+
+    public function getCountries(): array
```

Comment:
This method is really needed ?

What do you think to call directly `UnitedNationsBundle::getCountries()` in the setter : 
```php
if (!\in_array($country, array_keys(UnitedNationsBundle::getCountries()), true)) {
```